README.md

```diff
@@ -292,6 +292,37 @@ echo 'a';
 
 See also [example #3](examples).
 
+### addSignal()
+
+**Note: Signaling is only available on Unix-like platform, Windows isn't supported.**
```

Comment:
I think that it makes sense to not support Windows in the first step. :+1: 

However, note that others (https://docs.python.org/3/library/signal.html#signal.signal or https://github.com/nodejs/node-v0.x-archive/issues/1553) do provide limited, simulated signal handling supports on Windows. As such, I would suggest moving this note down a bit and make it more clear that this is not an interface limitations.

We should probably document the exception that we throw when signal handling is not supported. This should also cover Windows compatibility.